frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -180,7 +187,15 @@ export function AggregationPicker({
     }
 
     return sections;
-  }, [metadata, query, stageIndex, clauseIndex, operators, hasExpressionInput]);
+  }, [
+    metadata,
```

Comment:
I started thinking this useMemo is not needed here, we probably do not re-render list anyway